data/ksen21.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "Kartik Sen",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "front-end developer turned **Digital Nomad**. Tech enthusiast and Money hacks",
+  "avatar": "https://github.com/ksen21.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/imkartiksen",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Linkedn",
+      "url": "https://www.linkedin.com/in/kartik-sen-22300a137/",
+      "icon": "FaLinkednin"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/procodehelp/",
+      "icon": "FaInstagram"
+    }
```

Missing closing bracket here.
```suggestion
    }
  ]
```